### ISSUE TITLE
Lecture notes

### ISSUE THREAD

**Issue by VINX89** [wildtreetech/advanced-comp-2017#15]
It would be nice to have a scanned copy of the notes used during the lectures of May 1st and 8th (even a simple photo would work). Is that possible? Thank you.

**Comment by betatim** [comment]
Yes, I will take pictures of my notes and add them to the repository.
